bitrise.yml

```diff
@@ -334,6 +336,81 @@ workflows:
       bitrise.io:
         stack: osx-xcode-15.2.x
         machine_type_id: g2-m1.8core
+  determine_apps_affected:
+    steps:
+    - git-clone@6.2: {}
+    - script@1:
+        title: Determine if source is available
+        run_if: true
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -e
+
+            # Set IS_SOURCE_AVAILABLE based on the existence of BITRISE_SOURCE_DIR  (i.e. did the Git clone step succeed?)
+            IS_SOURCE_AVAILABLE=$([ -d "$BITRISE_SOURCE_DIR" ] && echo "true" || echo "false")
+            
+            envman add --key IS_SOURCE_AVAILABLE --value="$IS_SOURCE_AVAILABLE"
+            
+            echo "$IS_SOURCE_AVAILABLE"
+    - script@1:
+        title: Determine which application to base configuration
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -e
+
+            # Determine which application to base configuration
+            APPS_AFFECTED=$(git diff --name-only origin/main..."$BITRISE_GIT_COMMIT" | xargs -n1 dirname | sort | uniq)
+
+            # Initialize all application flags as "false" by default
+            BUILD_FIREFOX_IOS="false"
+            BUILD_FOCUS_IOS="false"
+            BUILD_SAMPLE_BROWSER_IOS="false"
+
+            # Check for specific applications and set flags accordingly
+            for app in $APPS_AFFECTED; do
```

Comment:
The files changed are ". firefox-ios" so we are in case "." where all variables are true, just to be sure that's on purpose for now, otherwise only firefox should be true according to the changes in this PR